README.md

```diff
@@ -1,13 +1,80 @@
-(ACK) service controller for ACM-PCA.
+(ACK) service controller for AWS Private Certificate Authority.
 
 Please [log issues][ack-issues] and feedback on the main AWS Controllers for
 Kubernetes Github project.
 
 [ack-issues]: https://github.com/aws/aws-controllers-k8s/issues
 
+## Resources Supported
+The ACK service controller for AWS Private Certificate Authority supports the following resources:
+- CertificateAuthority
+- Certificate
+- CertificateAuthorityActivation
+
+## Getting Started
+
+### Pricing
+Learn more about [AWS Private Certificate Authority Pricing][pricing].
+
+[pricing]: https://aws.amazon.com/private-ca/pricing
+
+### Samples 
+Go to the [samples directory][samples] and follow the README to create resources.
+
+[samples]: https://github.com/aws-controllers-k8s/acmpca-controller/tree/main/samples
+
+### Kubernetes Secrets
+The ACK service controller for AWS Private Certificate Authority uses Kubernetes Secrets to store certificate and certificate chains. Users are expected to create Secrets before creating Certificate and CertificateAuthorityActivation resources. As these resources are created, the Secrets will be injected with either the certificate or certificate chain. Users are responsible for deleting Secrets.
+
+#### Certificate Secret
+Before creating the Certificate resource, users must specify the namespace, name, and key of the Secret in the annotations of the resource, as shown below. If a namespace isn't specified, the namespace of the Certificate resource will be used. If a key isn't specified, the default key of `certificate` will be used.
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: certificate-secret
+  namespace: default
+data:
+  certificate: ""
+---
+apiVersion: acmpca.services.k8s.aws/v1alpha1
+kind: Certificate
+metadata:
+  name: my-certificate
+  annotations:
+    acmpca.services.k8s.aws/certificate-secret-namespace: default
+    acmpca.services.k8s.aws/certificate-secret-name: certificate-secret
+    acmpca.services.k8s.aws/certificate-secret-key: certificate
+...
+```
+
+#### CertificateChain Secret
+Before creating the CertificateAuthorityActivation resource, users must specify the namespace, name, and key of the Secret in the annotations of the resource, as shown below. If a namespace isn't specified, the namespace of the CertificateAuthorityActivation resource will be used. If a key isn't specified, the default key of `certificateChain` will be used.
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: certificate-chain-secret
+  namespace: default
+data:
+  certificateChain: ""
+---
+apiVersion: acmpca.services.k8s.aws/v1alpha1
+kind: CertificateAuthorityActivation
+metadata:
+  name: root-ca-activation
+  annotations:
+    acmpca.services.k8s.aws/chain-secret-namespace: default
+    acmpca.services.k8s.aws/chain-secret-name: certificate-chain-secret
+    acmpca.services.k8s.aws/chain-secret-key: certificateChain
+...
+```
+
```

Comment:
Add a blurb here on how to actually use this with the CertManager Plugin for PCA to start issuing certificates.

Reply:
would it be slightly confusing to add that? You can issue certificates with ACK if you want, there is just no management involved in the certificate issued via ACK.